File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/EventNotificationSample/src/com/hp/workpath/sample/eventnotificationsample/receiver/JobCompletedReceiver.java
package com.hp.workpath.sample.eventnotificationsample.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

import com.hp.workpath.sample.eventnotificationsample.util.PreferenceManager;

public class JobCompletedReceiver extends BroadcastReceiver {

    public static final String TAG = "[SAMPLE]" + "JobCompletedER";

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.i(TAG, "Job completed event received");
        PreferenceManager.saveEvent(context, intent.getAction(), null);
    }
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/EventNotificationSample/src/com/hp/workpath/sample/eventnotificationsample/receiver/JobCompletedReceiver.kt
package com.hp.workpath.sample.eventnotificationsample.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log
import com.hp.workpath.sample.eventnotificationsample.util.PreferenceManager

class JobCompletedReceiver : BroadcastReceiver() {
    companion object {
        val TAG = "[SAMPLE]JobCompletedER"
    }
    override fun onReceive(context: Context?, intent: Intent?) {
        Log.i(TAG, "Job completed event received")
        PreferenceManager.saveEvent(context, intent?.action, null)
    }
}